doc/release/README.tasks.rst

```diff
@@ -2,6 +2,12 @@
 Chapel programs, tasks, and threads
 ===================================
 
+.. contents::
```

Comment:
Can you remove this? I think it adds clutter, and is mostly served by the index on the left hand side